docs/release/Betaflight-4.5-Release-Notes.md

```diff
@@ -0,0 +1,284 @@
+Always test new firmware carefully and in a controlled environment.
+:::
+ 
+Do not use 4.3 or earlier dumps or Presets in 4.5.
+
+## 1. Cloud Build
+
+The cloud build system [introduced in 4.4](https://betaflight.com/docs/release/Betaflight-4.4-Release-Notes#1-cloud-build) retains the same user interface.  Updated board configurations should result in correct default hardware assignment.
+
+## 2. GPS
+
+The code connecting Betaflight to a GPS Module has been thoroughly overhauled.  
+
+When the FC boots, our UBLox code cycles through all available baud rates on the GPS Port until we connect to the module.  Then we instruct the module to change its baud rate to match the requested baud rate, which defaults to 57600, and we re-connect at that baud rate.  We then detect the 'class' of GPS module (M10, M8 etc) so that we know what kind of configuration requests it will respond to, and we re-configure it to send only the values we need, and stop it sending any other data. This ensures that the traffic on the serial port is the absolute minimum required for our purposes, and reduces CPU time.
+
+If the FC is connected to Configurator at boot time, we request the full satellite information list, so that we can populate the detailed satellite information list on the left side of Congfigurator's GPS tab.  Otherwise this information is not requested, because we do not require it while in flight, and it adds a lot of serial port traffic when enabled.
+
+The CPU cost and task timing for GPS data has been extensively reviewed and optimised.  Even so, GPS Rescue puts a huge load on a CPU.  For reliability it is best to use a 4k PID loop on all F4 processors, especially at 57600 baud.  More information about CPU load vs Baud Rate is available in the [GPS Rescue 4.5 documentation](https://betaflight.com/docs/wiki/archive/GPS-Rescue-v4-5).  The CLI `tasks` command may be used to check CPU usage and task over-runs when evaluating the impact of baud rate in relation to PID loop frequency. 
+
+The UBlox module 'class', and the baud rate it is actually connected at, may be checked with the CLI `status` command.
+
+The `GPS_CONNECTION` debug has been added, including a number of fields to help debug GPS task baud rates during the connection process, CPU cost, and ongoing execution times and intervals.
+
+Note that the GPS module must be powered up at the same time as the FC for it to be correctly configured.
+
+NMEA support is now very limited.  Using NMEA is not recommended.  Modern M10 GPS modules with a backup battery are highly recommended.  
+
+There should be no need for a user with an M8 or higher UBlox module to customise it in any way, e.g. with uCenter, unless it is somehow strangely locked and unresponsive to normal UBlox configuration commands.  They should essentially all work 'out of the box'.
+
+Thanks to @unit/freasy, ctzsnooze and SteveCEvans for this epic effort.
```

Comment:
Caution with the @, put to all our any

Reply:
removed them all, thanks for picking that up